packages/hamburger-react/src/Hamburger.tsx

```diff
@@ -1,54 +1,36 @@
-import React, { RefObject, useRef, useState } from "react";
-import { useClickOutside } from "@fremtind/jkl-react-hooks";
+import React from "react";
 
 interface Props {
     className?: string;
-    onClick?: (isActive: boolean) => void;
-    onClickOutside?: (isActive: boolean) => void;
-    enableClickOutside?: boolean;
-    initialIsActive?: boolean;
+    onClick?: () => void;
+    active: boolean;
     negative?: boolean;
```

Comment:
Kanskje vi kan benytte anledningen til å endre denne til å hete `inverted` nå som det uansett blir breaking change? Er fint om det er konsekvent i alle komponentene 😊

```suggestion
    inverted?: boolean;
```